modules/microshift-config-parameters-table.adoc

```diff
@@ -65,10 +65,55 @@ The following table explains {microshift-short} configuration YAML parameters an
+
+* The format must follow the Content-Type definition in RFC 1341: `Content-Type := type \"/\" subtype *[\";\" parameter]`.
+* The value of `Content-Type` can be one of the following types: application, audio, image, message, multipart, text, video, or a custom type preceded by `\"X-\"` and followed by a token defined as follows:
+* The token is a `string` of at least one character, and does not contain white spaces, control characters, or any of the characters in the `tspecials`set.
+* The `tspecials` set contains the characters `()\u003c\u003e@,;:\\\"/[]?.=`.
+* The subtype in Content-Type is also a token.
+* The optional parameters following the subtype are defined as `\n token \"=\" (token / quoted-string)`.
+* The `quoted-string`, as defined in RFC 822, is surrounded by double quotes and can contain white spaces plus any character except `\\`, `\"``, and `CR`. The `quoted-string` can also contain any single ASCII character if it is escaped by on the the following characters: `\\.",`.
+
+[NOTE]
+====
+Not all MIME types benefit from compression, but `HAProxy` uses resources to try to compress if instructed to. Generally speaking, text formats such as `html`, `ccs`, and `js` benefit from compression. The benefits to files that are already compressed, such as images, audio, and video, are likely not worth the CPU resources spent compressing these again.
+====
+//Q:Do we need to also give the pattern as a reference?
+
+|`ingress.httpEmptyRequestsPolicy`
+|`Respond` or `Ignore`
+|Default value is `Respond`. Describes how HTTP connections should be handled if the connection times out before a request is received. These connections typically come from a load balancer service's health probes or a web browser's speculative connections, such as a `preconnect`.
+* If the field is set to `Respond`, the ingress controller sends an "HTTP 400" or "408" response, logs the connection if access logging is enabled, and counts the connection in the appropriate metrics.
+* If the field is set to `Ignore`, the ingress controller closes the connection without sending a response, logging the connection, or incrementing metrics. Setting this field to `Ignore` might impede detection and diagnosis of problems or intrusions.
+
 |`ingress.listenAddress`
 |IP address, NIC name, or multiple
 |Value defaults to the entire network of the host. The valid configurable value is a list that can be either a single IP address or NIC name or multiple IP addresses and NIC names.
 
+|`ingress.logEmptyRequests`
+|`Log` or `Ignore`
+| Default value is `Log`. Specifies how connections on which empty requests are received are logged. These connections typically come from a load balancer service's health probes or a web browser's speculative connections, such as a `preconnect`. Logging typical requests might be undesirable, but requests can also be caused by network errors or port scans, in which case logging can be useful for diagnosing errors and detecting intrusion attempts.
```

Comment:
Please add "In addition, these requests may be caused by port scans, in which case logging empty requests may aid in detecting intrusion attempts. " thanks.